docs/administration-guide/secure-database-connections-with-ssl-certificates.md

```diff
@@ -0,0 +1,63 @@
+## Securing database connections using an SSL certificate
+
+If you'd like to connect your Metabase Cloud or self-hosted instance to a database, you can secure the connection using Secure Socket Layer (SSL) encryption with a certificate.
+
+Why you'd want to do this:
+
+- You're using Metabase Cloud and want to create a secure connection to a data warehouse that accepts a JDBC connection (e.g., PostgreSQL).
```

Comment:
I would say here: You're using Metabase Cloud and want to ensure the identity of the Data Warehouse you're connecting to (e.g., PostgreSQL, MySQL).